src/env-inl.h

```diff
@@ -356,6 +356,28 @@ inline IsolateData* Environment::isolate_data() const {
   return isolate_data_;
 }
 
+#if HAVE_INSPECTOR
+inline void Environment::ContextCreated(node::inspector::ContextInfo* info) {
+  contexts()->push_back(info);
+  if (inspector_agent()->IsStarted()) {
+    inspector_agent()->ContextCreated(info);
+  }
+}
+inline void Environment::ContextDestroyed(v8::Local<v8::Context> context) {
+  for (auto i = std::begin(*contexts()); i != std::end(*contexts()); ++i) {
+    auto it = *i;
+    if (it->context(isolate()) == context) {
+      delete it;
+      contexts()->erase(i);
+      if (inspector_agent()->IsStarted()) {
+          inspector_agent()->ContextDestroyed(context);
+      }
+      return;
+    }
+  }
+}
+#endif
```

Comment:
Can you write this as `#endif  // HAVE_INSPECTOR`?  Note the two spaces after `#endif`.